uberon.obo

```diff
@@ -113,6 +113,7 @@ synonymtypedef: scientific_name ""
 synonymtypedef: SENSU "taxonomic disambiguation"
 synonymtypedef: synonym ""
 synonymtypedef: SYSTEMATIC "a systematic synonym used as the base name for design patterns"
+synonymtypedef: vertebrate ""
```

Comment:
I also noticed this new synonym type. Is that right?

Reply:
seems dump. Can you figure out where it comes from form the edit file?

Reply:
yeah I saw it too, but it seems to come from a BSPO term (posterior_to) so I figured it was not on uberons side.